packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/validation.tsx

```diff
@@ -134,8 +134,10 @@ export const getMaxMin = (
           let limitMaxAmount = Number(pair.buyMax)
           let limitMaxChanged = false
           if (limits?.maxOrder) {
-            const buyMaxItem = Number(convertBaseToStandard('FIAT', limitMaxAmount))
-            const baseMaxLimitAmount = Number(limits.maxOrder)
+            const buyMaxItem = Number(limitMaxAmount)
+            const maxOrderBase = convertBaseToStandard('FIAT', limits.maxOrder, false)
```

Comment:
limits.maxOrder is always formatted amount so here we are putting it to base since all other amounts are minor string/base format